enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/core/replication/BenchmarkResult.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2002-2017 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.causalclustering.core.replication;
+
+@SuppressWarnings( "WeakerAccess" )
+public class BenchmarkResult
+{
+    public Long totalRequests;
+    public Long totalBytes;
+    public Long timeMillis;
```

Comment:
Iirc, these are all limitations put by the procedure system, because they use reflection and only support a limited set of types.